src/app/features/home/home.page.ts

```diff
@@ -197,9 +198,17 @@ export class HomePage {
 
   // eslint-disable-next-line class-methods-use-this
   async openCaptureClub() {
```

Comment:
```suggestion
  openCaptureClub() {
```
Keeping `async` keyword might be confusing since the function is not supposed to return Promise anymore.